test/Microsoft.AspNet.Hosting.Tests/testroot/wwwroot/TextFile.txt

```diff
@@ -0,0 +1 @@
+﻿A text file.
```

Comment:
Change its contents to indicate that is the intention of having that file there.